### PR TITLE
Retrieve logged level for content and display in about

### PR DESCRIPTION
**Pull request by AzariahAeroborn**
Draft PR for initial interface and criticism.  At a minimum, I want to clean up the About.tsx file further before this merges this way (and it looks like I may need to touch up the unit tests for the ActorUpdate synths as well)

### CODE REVIEW

File: src/parser/core/modules/About.tsx [hunk 1]
```diff
@@ -25,6 +29,7 @@ export default class About extends Analyser {
 	//		from: ...,
 	//		to: ...,
 	// }
+	supportedLevels = null
```
Comment (ackwell) [inline src/parser/core/modules/About.tsx]: hm. i'm not sure how to feel about explicitly mentioning "support". iiuc, we're not really trying to support synced content, but rather making sure synced content isn't missing data completely in the timeline.

**Review by ackwell — COMMENTED** [review]
general setup makes sense. i'm not sold on calling out support ranges explicitly, happy to discuss.

File: src/parser/core/modules/About.tsx (hunk 1 above)
Comment (AzariahAeroborn) [reply src/parser/core/modules/About.tsx]: Removed the supportedLevels support range.

**Comment by AzariahAeroborn** [comment]
I updated the way to access level to make it a getter on ActorResources -- while I was digging into this, found that my initial implementation besides being a pain to use also wound up crawling the actor updates a bunch (every time the ActorResources constructor got called) to retrieve the level property.  Getter seems to work as expected since the actorUpdate with the actor's level is going to be the first packet we receive for any actor in a given pull.

**Review by ackwell — APPROVED** [review]

**Pull request merged**